pkg/cc/config/config.go

```diff
@@ -180,6 +181,41 @@ func (c *Config) SaveCreds() error {
 	return nil
 }
 
+func (c *Config) SaveConfig(configPath string) error {
+
+	var tmp struct {
+		FileStoreRoot string        `json:"file_store_root"`
+		DefaultDomain string        `json:"default_domain"`
+		Logging       logger.Config `json:"logging"`
+		CA            []string      `json:"ca"`
+		Insecure      bool          `json:"insecure"`
+	}
+	tmp.FileStoreRoot = c.FileStoreRoot
+	tmp.DefaultDomain = c.DefaultDomain
+	tmp.Logging = c.Logging
+	tmp.CA = c.CA
+	tmp.Insecure = c.Insecure
```

Comment:
Do we need to write default values to the file, for the settings not provided by the user?